.github/workflows/unit.yml

```diff
@@ -1,15 +1,19 @@
-on: push
+# Run unit tests on any branch/fork pull request
+on:
+  pull_request
```

Comment:
not sure if worth specifying the `types` of pull_request events. See https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request and https://frontside.com/blog/2020-05-26-github-actions-pull_request/